<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DodajMotocyklDialog</class>
 <widget class="QDialog" name="DodajMotocyklDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Dodaj Motocykl Pojazd - dialog</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Marka:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="markaLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="modelLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cena:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cenaLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Rok:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="rokLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Przebieg:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="przebiegLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Moc:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mocLineEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DodajMotocyklDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DodajMotocyklDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
